File: reimi/frontend/reimi_app/lib/domain/value_objects/body_shape.dart
import 'package:freezed_annotation/freezed_annotation.dart';

enum BodyShape {
  @JsonValue('SLIM')
  slim,
  @JsonValue('SLENDER')
  slender,
  @JsonValue('NORMAL')
  normal,
  @JsonValue('MUSCULAR')
  muscular,
  @JsonValue('CHUBBY')
  chubby,
  @JsonValue('OVERWEIGHT')
  overweight,
}

File: reimi/backend/reimi_app/src/main/java/com/reimi/reimi_app/domain/model/profile/BodyShape.java
package com.reimi.reimi_app.domain.model.profile;

public enum BodyShape {
    SLIM("スリム"),
    SLENDER("やや細め"),
    NORMAL("普通"),
    MUSCULAR("筋肉質"),
    CHUBBY("ややぽっちゃり"),
    OVERWEIGHT("太め");

    private final String label;

    BodyShape(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }
}
